pygmt/src/text.py

```diff
@@ -15,6 +16,7 @@
 
 
 @fmt_docstring
+@deprecate_parameter("incols", "use_word", "v0.8.0", remove_version="v1.0.0")
```

Comment:
v1.0.0 is our target stable release for PyGMT. It definitely will take more times so it will be v0.10.0 after two minor releases.
```suggestion
@deprecate_parameter("incols", "use_word", "v0.8.0", remove_version="v0.10.0")
```